dart/lib/src/protocol/sentry_event.dart

```diff
@@ -262,6 +262,15 @@ class SentryEvent {
       json['exception'] = {
         'values': [exception.toJson()].toList(growable: false)
       };
+    } else if (stackTrace is SentryStackTrace) {
+      json['threads'] = {
+        'values': [
+          {
+            'id': 0,
+            'stacktrace': (stackTrace as SentryStackTrace).toJson(),
```

Comment:
wondering if its worth to have the thread list or at least a thread object for it, because the threads object has more fields and we could use the `Isolate.current` to populate them, example, thread name with `Isolate.current.debugName`, we can create a card for it, but not really necessary now